test-result-summary-client/src/Dashboard/Widgets/Graph/DayTrader7.jsx

```diff
@@ -0,0 +1,249 @@
+import React, { Component } from 'react';
+import {
+    HighchartsStockChart, Chart, XAxis, YAxis, Legend,
+    LineSeries, Navigator, RangeSelector, Tooltip
+} from 'react-jsx-highstock';
+import DateRangePickers from '../DateRangePickers';
+import { Checkbox } from 'antd';
+import BenchmarkMath from '../../../PerfCompare/lib/BenchmarkMath';
+import math from 'mathjs';
+import utils from './utils';
+
+const map = {
+    "Test_openjdk8_j9_sanity.perf_x86-64_linux_Liberty": "Test_openjdk8_j9_sanity.perf_x86-64_linux_Liberty",
+};
+let display = {
+    "Test_openjdk8_j9_sanity.perf_x86-64_linux_Libertym": true,
+};
+//currently no baseline runs are made we use hard coded baseline values for scaling graph
+let baseLineValue = 300000;
```

Comment:
To be consistent, it should be `baselineValue`. Similar issue in other places as well.

Reply:
If the value does not change, please use `const` not `let`. Please fix this issue for the rest of file